### PR TITLE
Prepare for 1.11.1 release

### PR DESCRIPTION
**Pull request by LordAro**
## Motivation / Problem
We should do a release

## Description
Cherry-picked all commits (2 needed manual edits, manually did the Romanian plural-deletion commit) & backported translations.
See final commit for changelog entries

## Limitations
~Still waiting on #9021 & #9047~
Untested.

## Checklist for review

Some things are not automated, and forgotten often. This list is a reminder for the reviewers.
* The bug fix is important enough to be backported? (label: 'backport requested')
* This PR affects the save game format? (label 'savegame upgrade')
* This PR affects the GS/AI API? (label 'needs review: Script API')
    * ai_changelog.hpp, gs_changelog.hpp need updating.
    * The compatibility wrappers (compat_*.nut) need updating.
* This PR affects the NewGRF API? (label 'needs review: NewGRF')
    * newgrf_debug_data.h may need updating.
    * [PR must be added to API tracker](https://wiki.openttd.org/en/Development/NewGRF/Specification%20Status)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+Fix #8956: Industry disaster news messages showed the wrong location (#8992)
+Fix: [Win32] Font glyphs of certain widths had broken rendering (#8990)
+Fix #8930: [Win32] Duplicate text input issue for systems using IME (#8976)
+Fix: [Network] Potential stale client entries in client list (#8959)
+Fix: Graphical issues when dragging measurement tooltips (#8951)
+Fix: [Fluidsynth] Use provided default soundfont if available (#8948, #8953)
+Fix #8935: [macOS] Crash on save (#8944)
+Fix #8922: Crash when selling shared vehicles with shared vehicle window open (#8926)
+Fix: Compiling on armhf (Raspberry Pi) (#8924)
+
```
Comment (TrueBrain) [inline changelog.txt:31]: Before your own OCD triggers next round: missing one newlines :) (should be 2 instead of 1, looking at the rest)

**Review by TrueBrain — DISMISSED** [review]
I assume you did a smoketest; I did not check anything about this PR other than that the changelog "looks good" :)

**Comment by michicc** [comment]
Hmm, title game doesn't load on web preview, but loads normally locally for me. Might just be something with the preview, as it works otherwise.

**Comment by TrueBrain** [comment]
> 
> 
> Hmm, title game doesn't load on web preview, but loads normally locally for me. Might just be something with the preview, as it works otherwise.

if I would be to guess, LZMA is not working in Emscripten. They were going to add the backport, but I don't think they did after all. So that would explain why the titlegame is not loading :D

**Comment by stormcone** [comment]
Would you consider also backporting #8908? Object and train station filters are new in 1.11.0, and perhaps some users would find the hotkeys useful.

**Comment by LordAro** [comment]
Sure, why not

**Review by michicc — APPROVED** [review]

**Pull request merged**